### DISCUSSION TITLE
How to follow redirects?

### DISCUSSION THREAD

**Discussion by Dronakurl** [mistweaverco/kulala.nvim#322] [Q&A] [answered]
```
###
GET http://localhost:8000/inspection/selection HTTP/1.1
accept: application/json
content-type: application/json
```
How can I follow redirects? I get this header, but no output. In curl, I use `--location-` to retrieve the result from the redirected endpoint. 
```
HTTP/1.1 301 Moved Permanently
date: Sun, 01 Dec 2024 15:19:22 GMT
server: uvicorn
content-length: 0
location: http://localhost:8001/inspection/selection
```

**Comment by Grueslayer** [comment, marked as answer]
You can add any curl options by adding these in the config:

https://kulala.mwco.app/docs/getting-started/configuration-options#additional_curl_options
